Plots/Bar/NCL_bar_11.py

```diff
@@ -0,0 +1,78 @@
+"""
+NCL_bar_11.py
+===============
+This script illustrates the following concepts:
```

Comment:
Do "changing the aspect ratio of a bar plot" and "setting the min/max value of the X and Y axis in a bar plot" still apply to the python version?

Reply:
The first one no longer applies but the second one does. I set the limits of the axes.